addon/services/stripe.js

```diff
@@ -0,0 +1,229 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend(Ember.Evented, {
+
+  /**********************************
```

Comment:
@vladucu lots of duplicate code in `addon/services/stripe` and `addon/components/stripe-checkout`, I wonder if we can share most of it between the two. Maybe a mixin called `stripe-config-attributes` or something.